frontend/pages/artist/_itemId/index.vue

```diff
@@ -142,8 +127,11 @@ import { sanitizeHtml } from '~/utils/html';
 import { getImageInfo, getBlurhash, ImageUrlInfo } from '~/utils/images';
 import { getItemDetailsLink, isValidMD5 } from '~/utils/items';
 import { pageStore, authStore } from '~/store';
+import ArtistTab from '~/components/Layout/Artist/ArtistTab.vue';
+import { msToTicks } from '~/utils/time';
 
 export default Vue.extend({
+  components: { ArtistTab },
```

Comment:
```suggestion
import { msToTicks } from '~/utils/time';

export default Vue.extend({
```

@BrianCArnold Nuxt automatically imports components for us, so we don't need this.
Feel free to commit this suggestion through GitHub UI or do it locally if you want to squash it with another commit or place it somewhere else in your history.

Reply:
Thanks, years of experience with Angular, but brand new to Vue.